functorch/functorch/_src/eager_transforms.py

```diff
@@ -855,6 +855,9 @@ def jacfwd(func: Callable, argnums: argnums_t = 0, has_aux: bool = False):
             the function to be differentiated and the second element is
             auxiliary objects that will not be differentiated.
             Default: False.
+        randomness(str): Flag indicating what type of randomness to use.
+            See :func:`vmap`. Allowed: "different", "same", "error".
```

Comment:
nit: see :func:`vmap` for more details

Could you check that the docs for this build before merging it? Since we don't have a docs build set up in pytorch/pytorch yet

Reply:
Edited and checked docs. Link goes to vmap correctly

<img width="717" alt="Screen Shot 2022-09-06 at 4 24 25 PM" src="https://user-images.githubusercontent.com/17888388/188733544-078f712a-b18a-49f8-8f5f-8c774ed8a2fb.png">